style.css

```diff
@@ -511,3 +519,245 @@ input:valid {
 .img-bar {
   justify-content: center;
 }
+
+@media (min-width: 768px) {
+  .container {
+    width: 100%;
+    height: 3074px;
+  }
+
+  body {
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    overflow-x: hidden;
+  }
+
+  .nav img {
+    display: none;
+  }
+
+  .desk {
+    display: block;
+    margin-top: 0;
+  }
+
+  .desk ul {
+    display: flex;
+    justify-content: center;
+    list-style: none;
+    width: 97%;
+    height: 72px;
+    margin-top: -10px;
+    margin-bottom: 20px;
+    align-items: center;
+    gap: 10px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+  }
```

Comment:
- [ ] Kindly consider Implement the changes suggested by the previous reviewer as I can see that you have not implemented this change.